src/com/facebook/buck/maven/ArtifactConfig.java

```diff
@@ -0,0 +1,102 @@
+package com.facebook.buck.maven;
+
+import com.google.common.base.Charsets;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.annotations.SerializedName;
+
+import org.kohsuke.args4j.Argument;
+import org.kohsuke.args4j.CmdLineException;
+import org.kohsuke.args4j.CmdLineParser;
+import org.kohsuke.args4j.Option;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ArtifactConfig {
+
+  private static class CmdLineArgs {
+    @Argument(usage="Artifacts to resolve", metaVar="artifact")
+    public List<String> artifacts = new ArrayList<>();
+
+    @Option(name="-repo", usage="Buck repository root")
+    public String buckRepoRoot;
```

Comment:
It's not clear from this if it's meant to be Buck's root or the project's Buck root (as in where the `.buckconfig` lives)